app/components/framework/CompactAttributeInput/WidgetWrapper/index.tsx

```diff
@@ -42,7 +42,7 @@ function WidgetWrapper(props: Props) {
                 className={_cs(headerClassName, styles.header)}
                 // FIXME: use strings
             >
-                {title ?? 'Unnamed'}
+                {title}
```

Comment:
Why remove default title?
This is very useful when we are creating a new widget